src/assets/scripts/client/math/flightMath.js

```diff
@@ -168,7 +169,7 @@ const _calculateCourseChangeInRadians = (currentHeading, nominalNewCourse) => {
  * @param aircraft {AircraftModel}
  * @param fix
  */
-export const calculateTurnInitiaionDistance = (aircraft, currentWaypointPosition) => {
+export const calculateTurnInitiationDistance = (aircraft, currentWaypointPosition) => {
```

Comment:
Spelling is hard, thanks for updating this. Could you also update the docblock here too?

Reply:
done